Public/Src/Engine/Dll/Tracing/Log.cs

```diff
@@ -1703,14 +1703,29 @@ public abstract partial class Logger
             Message = "{ShortProductName} failed to run because only 'case-insensitive' file-systems are currently supported on non-windows hosts.")]
         public abstract void ErrorCaseSensitiveFileSystemDetected(LoggingContext context);
 
+        public void BusyOrUnavailableOutputDirectories(LoggingContext context, string objectDirectoryPath, string exception)

```

Comment:
>BusyOrUnavailableOutputDirectories [](start = 20, length = 34)

Didn't see any callers added to this method...

Reply:
It's a bit of a trick. This new method has the same method signature of the old one. So callers that were previously logging the one message are now logging the shorter one on the console and then longer one for the log file.

---
In reply to: [271098653](https://github.com/Microsoft/BuildXL/pull/68#discussion_r271098653) [](ancestors = 271098653)